### ISSUE TITLE
Is there a way to show the utilization of each GPU?

### ISSUE THREAD

**Issue by hlc1209** [fossfreedom/indicator-sysmonitor#134]
First of all, thank you for creating this fantastic tool. I was wondering if there is a way to display the utilization of each GPU?

**Comment by fossfreedom** [comment]
do you mean this? https://github.com/fossfreedom/indicator-sysmonitor/pull/116

**Comment by hlc1209** [comment]
> do you mean this? #116

Exactly. This commit looks great, but I'm still hoping for official support.

**Comment by fossfreedom** [comment]
I'm happy to merge that - but I need someone to resolve the conflict and check that the suggested PR works.  I don't have nvidia to test with so I'm not going to merge blind here.

**Comment by hlc1209** [comment]
> I'm happy to merge that - but I need someone to resolve the conflict and check that the suggested PR works. I don't have nvidia to test with so I'm not going to merge blind here.

No problem, I'd love to help with that. I will try to merge and test it on my server, and let you know the results.
